Add how to enable couch2pg postgres ingestion for CMA (#1051)
medic/cht-docs: content/en/apps/guides/hosting/monitoring/setup.md

### Additional Configuration

#### couch2pg Data

> With the [release of 1.1.0](https://github.com/medic/cht-monitoring/releases/tag/1.1.0), CMA now supports easily ingesting [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}}) data read in from a Postgres database.  While 1.1.0 only currently supports the couchd2pg backlog metric, more queries will be added over time.

Reviewer: ```suggestion
With the [release of 1.1.0](https://github.com/medic/cht-monitoring/releases/tag/1.1.0), CMA now supports easily ingesting [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}}) data read in from a Postgres database.
```

IMHO it is okay not to mention this. I expect we will add more queries very soon and we would have to remember to come back here and remove this then....

1. Copy the two example config files so you can add the correct contents in them.  Do so by running this code:
   
   ```shell
   cd ~/cht-monitoring
   cp exporters/postgres/postgres-instances.example.yml exporters/postgres/postgres-instances.yml
   cp exporters/postgres/postgres_exporter.example.yml exporters/postgres/postgres_exporter.yml
   ```
2. Edit `postgres-instances.yml` you just created and add your target postgres connection URL along with the proper root URL for your CHT instance as the label value. For example, if your postgres server was `db.example.com` and your CHT instance was `cht.example.com` the config would be:
   ```yaml
   - targets: [db.example.com:5432/cht]
     labels:
       cht_instance: cht.example.com
   ```

> 3. Edit `postgres_exporter.yml` so that the `auth_modules` object for your Postgres instance has the proper username and password. Using our `db.example.com` example from above, this might look like assuming a password of `super-secret` and a username of `pg_user`:

Reviewer: ```suggestion
3. Edit `postgres_exporter.yml` so that the `auth_modules` object for your Postgres instance has the proper username and password. Using our `db.example.com` example from above and assuming a password of `super-secret` and a username of `pg_user`, the config would be:
```
